.gitlab-ci.yml

```diff
@@ -73,8 +76,40 @@ stages:
     - ${EXEC_PREFIX} ${RESOURCES} scripts/gitlab/build_and_test.sh --test-only
     - echo -e "section_end:$(date +%s):tests\r\e[0K"
 
+
+.build_blueos_3_ppc64le_ib_p9_script:
```

Comment:
We could certainly improve that.
For example, I _think_ the "sections_starts" that delimitate foldable parts of the log, could be embedded in the script. But since in was not critical, I never took the time to test it.

Reply:
It's fine.. just grumbling that gitlab doesnt support the matrix functionality that most new CI programs do. Let me be grumpy @adrienbernede !